.github/workflows/run.yml

```diff
@@ -33,14 +33,22 @@ jobs:
       shell: bash -el {0}
       run: |
         source /opt/SIRF-SuperBuild/INSTALL/bin/env_sirf.sh
-        curl -fsSL https://raw.githubusercontent.com/SyneRBI/PETRIC/main/petric.py > petric.py
```

Comment:
we need this line to overwrite any users' changes as per the README

> Any modifications to `petric.py` are ignored.

Reply:
yes all submission repos are a fork of this one, so they all use this `run.yml`.

- pro: we all use the same `petric.py`
- con: it's impossible for organisers to test changes to `petric.py` in CI
  + I've had to manually test any change(s)